src/project_types/sequence_section.rs

```diff
@@ -12,37 +12,23 @@ use project_types::User;
 #[derive(Clone, Debug, PartialEq, Eq, RustcEncodable, RustcDecodable)]
 pub struct SequenceSection {
     pub seq_name: String,
-    pub index: u32,
+    pub index: u32,  // Starts at 1
+    pub path: String,
     pub num_frames: u32,
     pub editor: Option<User>,
```

Comment:
I think it's nice to be able to load a sequence section and know all of the data pertaining to it. However, I agree with you that it is redundant, ultimately not trustworthy (since keeping it in sync is hard to guarantee), and we could do without it. Let's remove for now. If it turns out it would be really useful, then we can always add it back in.